### DISCUSSION TITLE
Wrong version for babel/code-frame in babel/core

### DISCUSSION THREAD

**Discussion by tagarwal84** [babel/babel#12744] [Q&A] [answered]
Today wrong version of dependency pushed in babel/core 
there are no such versions present.
@babel/code-frame is 7.10.4 not 7.12.13

**Comment by tagarwal84** [comment, marked as answer]
Got it now the code is pushed for babel/code-frame too. some how it took 3 hours to present in npm repo.

**Reply by nicolo-ribaudo** [reply]
Sometimes npm is slow to show published packages 🤷
